force-app/main/default/objects/Session_Registration__c/fields/JunctionID__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>JunctionID__c</fullName>
+    <caseSensitive>false</caseSensitive>
+    <description>An id field for to detect if an attendee already registered to session.</description>
```

Comment:
@bmtemakina I believe description of this field is grammatically incorrect, please check it.